django/contrib/postgres/fields/ranges.py

```diff
@@ -92,25 +95,62 @@ def set_attributes_from_name(self, name):
 
     def value_to_string(self, obj):
         value = self.value_from_object(obj)
+        base_field = self.base_field
+
         if value is None:
             return None
+
+        if isinstance(value, (list, tuple)):
+            bounds = getattr(self, 'default_bounds', CANONICAL_RANGE_BOUNDS)
+            value = self.range_type(value[0], value[1], bounds=bounds)
```

Comment:
This change is not related with the PR. Serialization wasn't working for models with list or tuples for range fields.